shared/wallets/send-form/asset-input/asset-input-advanced.tsx

```diff
@@ -469,22 +469,25 @@ const styles = Styles.styleSheetCreate(
         flexShrink: 0,
       },
       pickAssetButton: Styles.platformStyles({
-        common: {
-          width: Styles.globalMargins.xlarge * 3,
-        },
         isMobile: {
+          //    width: Styles.globalMargins.xlarge * 3,
           //    paddingTop: Styles.globalMargins.tiny,
         },
       }),
       // We need this to make the PickAssetButton on top of other stuff so amount
       // error can extend below it.
-      pickAssetButtonOverlayInner: {position: 'absolute', right: 0, top: 0},
-      pickAssetButtonOverlayOuter: {position: 'relative'},
+      pickAssetButtonOverlayOuter: {
+        flexShrink: 0,
+        position: 'relative',
```

Comment:
Can remove this I believe

Reply:
just checking did you test with this removed?

Reply:
Yeah I toggled it off in the devtools and it was fine. We left it as `position: "relative"` after removing the absolute positioning, so it was extra. Nothing is positioned absolutely anymore.